usr/share/rear/verify/PPDM/default/400_check_and_wait_for_valid_lockbox.sh

```diff
@@ -0,0 +1,69 @@
+# check for valid lockbox
+
+# ddfsadmin lockbox -query
+
+# success return code 0 and this output
+# DD_IP           Storage Unit                                                     DD_USERNAME          DD_Config_Type
+# 192.168.1.30    Linux-ppdm-01-f3392/PLCTLP-580da36a-9e5d-4bc3-a438-d7d43cf34da4  Linux-ppdm-01-f3392  PROTECTION
+
+# failure return code also 0 and this or other output
+# 2024-01-16T15:30:51.103Z Error retrieving password from lockbox
+
+function ppdm_get_config_from_lockbox {
+    # Note: We replace Storage Unit with Storage_Unit so that our read call below can successfully split on space characters :-)
+    local res=$(ddfsadmin lockbox -query 2>&1 | sed -e 's/Storage Unit/Storage_Unit/g') _h1 _h2 _h3 _h4
+    (($? > 0)) && Error "Could not query lockbox:$LF$res"
```

Comment:
this won't work because the exit code from the previous line is the exit code from `local`, not from `ddfsadmin`.
Suggestion:
```
    local res
    res=$(ddfsadmin lockbox -query 2>&1 | sed -e 's/Storage Unit/Storage_Unit/g') _h1 _h2 _h3 _h4 || Error "Could not query lockbox:$LF$res"
```